### PR TITLE
[CR-88]: Improve error logging in analyze stage and other analyze improvements.

### PR DESCRIPTION
**Pull request by xun-cockroachlabs**
1. Hit parsing errors when analyzer processing the raw result files. It just gave raw errors without giving any additional information on error file, error line, offending column information, which makes debugging pretty hard. Add the relevant info will made debugging on the above errors much easier.
2. Because TPCC invalid files will cause parsing errors, need to also add change to exclude them from the processing.

Release note: none

### CODE REVIEW

File: cmd/generate.go [hunk 1]
```diff
@@ -93,6 +93,11 @@ function create_cluster() {
   roachprod create "$CLUSTER" -n $NODES --lifetime "{{.Lifetime}}" --clouds "$CLOUD" \
     --$CLOUD-machine-type "{{.MachineType}}" {{.NodeEastLocation}} {{.EvaledArgs}} {{.UsEastAmi}} \
     --label {{.Usage}}
+  
+  echo "---- start of cpu info for cluster $CLUSTER ----"
+  roachprod run "$CLUSTER":1 -- cat /proc/cpuinfo
+  echo "---- end of epu info for cluster $CLUSTER ----"
+
```
Comment (miretskiy) [inline cmd/generate.go:100]: We probably don't need this? I thought that cpu info  output was added recently.

File: cmd/generate.go [hunk 2]
```diff
@@ -548,4 +563,4 @@ func generateCloudScripts(cloud CloudDetails) error {
 	}
 
 	return nil
-}
+}
```
Comment (miretskiy) [inline cmd/generate.go:566]: nit: add a \n please

**Review by miretskiy — APPROVED** [review]

File: cmd/generate.go (hunk 1 above)
Comment (xun-cockroachlabs) [reply cmd/generate.go]: This should be auto merged when I rebase master, but I will revert it.

File: cmd/generate.go (hunk 2 above)
Comment (xun-cockroachlabs) [reply cmd/generate.go]: done.

**Comment by xun-cockroachlabs** [comment]
TFTR!

**Pull request merged**